core/test/src/test_rom.cpp

```diff
@@ -1,3 +1,4 @@
+#include "ippu_helpers.h"
```

Comment:
Why is this required here all of a sudden?

Reply:
Needed in order for gtest to find PrintTo() and use the same definition everywhere.